geode-core/src/main/java/org/apache/geode/internal/cache/LatestLastAccessTimeMessage.java

```diff
@@ -41,17 +43,20 @@
   private int processorId;
   private String regionName;
   private K key;
+  private boolean isInvalidate;
```

Comment:
We could handle this from the receiver point; by not returning the destroy/invalid time to the requester.

Reply:
done